src/fal/cli/dbt_model_generator.py

```diff
@@ -0,0 +1,132 @@
+import ast
+import astor
+import re
+from typing import List, Union
+from pathlib import Path
+from more_itertools import flatten
+
+from faldbt.project import FalDbt
+
+
+SQL_MODEL_TEMPLATE = """
+{{ config(materialized='ephemeral') }}
+/*
+FAL_GENERATED
+
+Script dependencies:
+
+__deps__
+
+*/
+
+SELECT * FROM {{ target.schema }}.{{ model.name }}
+"""
+
+
+def generate_python_dbt_models(fal_dbt: FalDbt):
```

Comment:
Why don't we generate python dbt models before initiating `fal_dbt`? All we need is a project dir.